eng/versioning/version_client.txt

```diff
@@ -8,6 +8,7 @@ com.azure:azure-ai-formrecognizer;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-ai-textanalytics;1.0.0;1.1.0-beta.1
 com.azure:azure-core;1.5.1;1.6.0-beta.1
 com.azure:azure-core-amqp;1.2.0;1.3.0-beta.1
+com.azure:azure-core-experimental;1.0.0-beta.1;1.0.0-beta.1
```

Comment:
This package should forever remain a beta as we don't want folks taking shipping dependencies on it. Do we have any great way to help enforce that?

Reply:
Not that I know of currently, @JimSuplizio do you have anything?

Reply:
@alzimmermsft @weshaggard 
There's nothing preventing anyone from changing the version of any library and shipping it so long as the version passes the format check and they have approval permissions. There's never been a "this library must remain beta" requirement before.

Reply:
OK @alzimmermsft @srnagar please try to keep an eye our to try and prevent this from shipping a GA as it will defeat the entire purpose of it if we do.